_includes/docs/latest/resource-coverage.md

```diff
@@ -8,11 +8,14 @@
 | [Bucket Access Controls](https://cloud.google.com/storage/docs/json_api/v1/bucketAccessControls#resource) |
 | [Cloud APIs Configuration](https://cloud.google.com/apis/) |
 | [Cloud Audit Logging Configuration](https://cloud.google.com/iap/docs/audit-log-howto) |
+| [Cloud Dataproc Clusters](https://cloud.google.com/dataproc/docs/reference/rest/v1/projects.regions.clusters) |
 | [Cloud IAM Grantable Roles](https://cloud.google.com/iam/reference/rest/v1/roles/queryGrantableRoles) |
 | [Cloud IAM Organization Roles](https://cloud.google.com/iam/reference/rest/v1/organizations.roles) |
 | [Cloud IAM Project Roles](https://cloud.google.com/iam/reference/rest/v1/projects.roles) |
 | [Cloud Pub/Sub](https://cloud.google.com/pubsub/docs/access-control) |
+| [Cloud Pub/Sub Subscriptions](https://cloud.google.com/pubsub/docs/reference/rest/v1/projects.subscriptions) |
 | [Cloud SQL](https://cloud.google.com/sql/docs/mysql/admin-api/v1beta4/instances#resource) |
+| [Cloud SQL Instances](https://cloud.google.com/sql/docs/mysql/admin-api/v1beta4/instances) |
```

Comment:
You should also remove the `Cloud SQL` line, right above this one.  they are the same.

Reply:
Removed.